src/System.Security.Cryptography.Cng/tests/ImportExportTests.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.Win32.SafeHandles;
+using Test.Cryptography;
+using Xunit;
+
+namespace System.Security.Cryptography.Cng.Tests
+{
+    public static class ImportExportTests
+    {
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public static void TestImportExport1()
```

Comment:
This test feels like it's at least three things:
1) key.Handle.Dispose() doesn't affect reading properties (maybe this isn't important, but I can't think of why else it's the first thing done)
2) That the default import results in all those properties being what you expect
3) That Export produces the same answer that Import did.

Can you please break them up into three named tests?

And not that it belongs here, but somewhere we should have a test that does
```C#
SafeNCryptKeyHandle keyHandle1 = key.Handle;
SafeNCryptKeyHandle keyHandle2 = key.Handle;

Assert.NotSame(keyHandle1, keyHandle2);
```